lib/licenses.nix

```diff
@@ -445,6 +445,13 @@ lib.mapAttrs mkLicense ({
     fullName = "curl License";
   };
 
+  defold = {
+    fullName = "Defold License";
```

Comment:
Given this will only ever be used by one package, maybe it'd make more sense to inline it?

Reply:
Alright, I've inlined it.